webpack.config.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js?$/,
-        exclude: /(node_modules)/,
+        test: /\.(js|jsx)?$/,
+        exclude:path.resolve(__dirname, "node_modules"),
```

Comment:
The previous exclude had the effect of excluding this directory when it was checked out (the submodule checkout is located at `src/node_modules/solr-faceted-search-react`).  This rule is stricter in that it only excludes the node_modules in the current directory.